@navikt/core/react/src/button/Button.tsx

```diff
@@ -113,41 +92,33 @@ export const Button: OverridableComponent<ButtonProps, HTMLButtonElement> =
         <Component
           {...(Component !== "button" ? { role: "button" } : {})}
           {...filterProps}
-          ref={mergedRef}
+          ref={ref}
           onKeyUp={composeEventHandlers(filterProps.onKeyUp, handleKeyUp)}
           className={cl(
             className,
             "navds-button",
             `navds-button--${variant}`,
             `navds-button--${size}`,
             {
-              "navds-button--loading": widthOverride,
+              "navds-button--loading": loading,
               "navds-button--icon-only": !!icon && !children,
-              "navds-button--disabled": disabled ?? widthOverride,
+              "navds-button--disabled": disabled ?? loading,
             },
           )}
-          style={{
-            ...style,
-            width: widthOverride,
-          }}
-          disabled={disabled ?? widthOverride ? true : undefined}
+          style={style}
```

Comment:
I think we can remove `style={style}` here, and remove `style` from the destructuring, so that it ends up in `rest`.

Reply:
Good catch!